src/main/java/Duke.java

```diff
@@ -1,45 +1,71 @@
 import java.util.Scanner;
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println(logo);
-        Init.greet();
-        //echo & exit
-        boolean flag = true;
+        showWelcomeMessage();
+
         TaskManager tasksList = new TaskManager();
-        while(flag){
+        while(true){
```

Comment:
Make sure to adhere to the Java Coding Standard. Note the spaces "while (true) {"